CHANGES_NEXT_RELEASE

```diff
@@ -1,2 +1,3 @@
+- [cygnus-commons] Check TextUnrestricted type to escape character ' as '' (#2125)
```

Comment:
Maybe something about this should be done in documenation?

Not sure in which .md. Do we have one about NGSIHandler? (in a similar way we have per-sink documents)

Reply:
done in https://github.com/telefonicaid/fiware-cygnus/pull/2128/commits/f4bdb5e0a8dd52f79fd43f8bcd958deb9d9f3cb5